common/src/main/java/tachyon/heartbeat/HeartbeatScheduler.java

```diff
@@ -63,19 +66,23 @@ public static void addTimer(ScheduledTimer timer) {
   public static synchronized void removeTimer(ScheduledTimer timer) {
     Preconditions.checkNotNull(timer);
     sLock.lock();
-    sTimers.remove(timer.getThreadName());
-    sLock.unlock();
+    try {
+      sTimers.remove(timer.getThreadName());
+    } finally {
+      sLock.unlock();
+    }
```

Comment:
For these, is there a point in doing `try...finally`, since any exception thrown will be an unchecked exception (which will probably crash the program anyway)?